packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionWithChoicesAndAutocompleteMixin.ts

```diff
@@ -32,15 +36,17 @@ export class ApplicationCommandOptionWithChoicesAndAutocompleteMixin<T extends s
 
 		validateChoicesLength(choices.length, this.choices);
 
-		for (const { name, value } of choices) {
+		for (const { name, name_localizations, value } of choices) {
 			// Validate the value
 			if (this.type === ApplicationCommandOptionType.String) {
 				stringPredicate.parse(value);
 			} else {
 				numberPredicate.parse(value);
 			}
 
-			this.choices!.push({ name, value });
+			validateNameLocalizationsMap(name_localizations);
```

Comment:
L31 does `choicesPredicate.parse(choices);`, where `choicesPredicate` includes validation for the `name_localizations` property.

Doing it again is unnecessary.